api/src/main/java/org/openmrs/hl7/impl/HL7ServiceImpl.java

```diff
@@ -393,6 +393,18 @@ public HL7InError getHL7InErrorByUuid(String uuid) throws APIException {
 		return dao.getHL7InErrorByUuid(uuid);
 	}
 	
+	/** Log an error when the user with the details idNumber, familyName and givenName cannot be found
```

Comment:
You could take a look at classes in the JDK to see how methods are described in the javadoc.